client/src/views/Profile/Card/index.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="profile-card">
+      <div class="user-avatar">
+        <img :src="profile.avatar_url" alt="avatar"/>
+        <i v-if="profile.counters.views > 100" class="fas fa-star" title="More than 100 views"></i>
```

Comment:
Do not use magic numbers, please move this into separate computed property and declare 100 as variable.